engineering-feedback/feedback-examples.md

```diff
@@ -0,0 +1,54 @@
+# Engineering Feedback Examples
+
+The following are real world examples of Engineering Feedback that have led to product improvements and unblocked customers.
+
+## Windows Server Container support for Azure Kubernetes Service
+
+### Content
+
+The Azure Kubernetes Service should have first class Windows container support so solutions that require Windows workloads can be deployed on a wildly popular container orchestration platform. The need was to be able to deploy Windows Server containers on AKS the managed Azure Kubernetes Service. According to this FAQ (and in parallel confirmation) it is not available yet: https://docs.microsoft.com/en-us/azure/aks/faq#can-i-run-windows-server-containers-on-aks
```

Comment:
Should we make the link be inline within the words "this FAQ"?

Reply:
In my second commit I'd made the link in "not available yet" so that's a little outdated, but it does make more sense to move the link to 'this FAQ' indeed, will fix